src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,20 @@
-export const UserInfo = () => {};
+import './UserInfo.scss';
+
+import { User } from '../../types';
+
+type Props = {
+  user: User;
```

Comment:
Consider renaming `user` prop to `userInfo` for better readability and consistency. 🤓